packages/text/src/markdown/compare.ts

```diff
@@ -34,13 +34,20 @@ export function calcSørensenDiceCoefficient (a: string, b: string): number {
  * Perform markdown diff/comparison to understand do we have a major differences.
  */
 export function isMarkdownsEquals (source1: string, source2: string): boolean {
-  const lines1 = source1
-    .split('\n')
-    .map((it) => it.trimEnd())
-    .join('\n')
-  const lines2 = source2
-    .split('\n')
-    .map((it) => it.trimEnd())
-    .join('\n')
+  const normalizeLineEndings = (str: string) =>
```

Comment:
If comments will not be supported we will potentially break description on github, let's support html/comments first and then add a trim of lines >2 into 1 into comparison, but not allow without line and with line be the same.

Reply:
Just fyi: even with this change description with comment and without comment will not be equal.